api/package.json

```diff
@@ -14,6 +14,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@aws-sdk/client-lambda": "^3.16.0",
+    "@aws-sdk/credential-provider-node": "^3.16.0",
     "axios": "^0.21.1",
     "cors": "^2.8.5",
     "docdash": "^1.2.0",
```

Comment:
unrelated to this PR, but `docdash` can be moved into `devDependencies` as its not needed for running the server